dbt/models/default/default.vw_pin_sale.sql

```diff
@@ -217,16 +207,15 @@ mydec_sales AS (
             ) > 0 AS sale_filter_ptax_flag,
             COUNT() OVER (
                 PARTITION BY line_1_primary_pin, line_4_instrument_date
-            ) AS num_single_day_sales,
-            year_of_sale
```

Comment:
Rather than removing this, I'd use it as your source of mydec sale year. It's constructed using the instrument date during ingest. No reason to introduce additional calculations.